tenable/io/v3/was/scans/api.py

```diff
@@ -165,40 +169,275 @@ def launch(self, config_id: UUID) -> None:
+                    ...                 ('test', 'oper', '2')
+                    ...             ),
+                    ...     'and', ('test', 'oper', 3)
+                    ... )
+                    >>> {
+                    ...  'or': [{
+                    ...      'and': [{
+                    ...              'value': '1',
+                    ...              'operator': 'oper',
+                    ...              'property': '1'
+                    ...          },
+                    ...          {
+                    ...              'value': '2',
+                    ...              'operator': 'oper',
+                    ...              'property': '2'
+                    ...          }
+                    ...      ]
+                    ...  }],
+                    ...  'and': [{
+                    ...      'value': '3',
+                    ...      'operator': 'oper',
+                    ...      'property': 3
+                    ...  }]
+                    ... }
+
+                As the filters may change and sortable fields may change over
+                time, it's highly recommended that you look at the output of
+                the :py:meth: `tio.v3.definitions.was.scans()`
+                endpoint
+                to get more details.
+            limit (int, optional):
+                Number of objects to be returned in each request.
+                Default and maximum limit is 200.
+            offset (int, optional):
+                The pagination offset to use when requesting the next page of
+                results.
+            num_pages (int, optional):
+                The total number of pages to request before stopping the
+                iterator.
+            return_resp (bool, optional):
+                If set to true, will override the default behavior to return
+                an iterable and will instead return the results for the
+                specific page of data.
+            return_csv (bool, optional):
+                If set to true, it will return the CSV response or
+                iterable (based on return_resp flag). Iterator returns all
+                rows in text/csv format for each call with row headers.
+            iterator_cls:
+                If specified, will override the default iterator class that
+                will be used for instantiating the iterator.
+            schema_cls:
+                If specified, will override the default Search schema class
+                that will be used for validation.
+
+        Returns:
+
+            Iterable:
+                The iterable that handles the pagination for the job.
 
-    # TODO: Requires search iterator
-    def search_vulnerabilities(self, id: UUID, **kwargs) -> None:
+            requests.Response:
+                If ``return_json`` was set to ``True``, then a response
+                object is instead returned instead of an iterable.
```

Comment:
Can you make sure, if return_json returns requests.Response object instead of dict when its set to true ?